starcheck/utils.py

```diff
@@ -59,8 +59,9 @@ def get_data_dir():
 def _make_pcad_attitude_check_report(kwargs):
     try:
         return make_pcad_attitude_check_report(**de_bytestr(kwargs))
-    except Exception as err:
-        perl.warning("Error running dynamic attitude checks {}\n".format(err))
```

Comment:
The `perl` reference was giving a NameError, not defined. Even if it were working, printing the error message (expecting str, bytes not NoneType) without a stack trace was useless for debugging.

 I was missing the ATTITUDE history file and perl helpfully pushed onward so that I later got a TypeError.

Reply:
That's fine.  I like the incremental improvements.  Should the traceback below also just `raise` like the ccd_temp_wrapper error handling?